src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,21 @@
-export const CommentList = () => (
-  <>Put the list here</>
-);
+import commentsFromServer from '../../api/comments.json';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+export const CommentList = ({ post }) => {
+  const comments = commentsFromServer
+    .filter(comment => comment.postId === post.id);
```

Comment:
you should pass the comments here as prop instead of the whole post, it must fix some tests